packages/fx-core/src/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import * as path from "path";
+
+
+export * from "./common";
+export * from "./plugins";
+export * from "./core";
+
+
+export function getTemplatesFolder(){
+    return path.join(__dirname, "../templates");
+}
+
+export function getResourceFolder(){
+    return path.join(__dirname, "../resource");
+}
```

Comment:
Why put these two functions in index.ts?

Reply:
to make sure the relative path of resource and templates are the same in development mode and the webpack mode